### PR TITLE
3 regex's (STDERR, STDIN, Python) to parse yt-dlp download progress, for live display in "Tasks" view

### PR DESCRIPTION
**Pull request by holta**
This PR is a proposed alternative to:

- PR #91
- PR #92 

Please see Lines 57-67 of `scripts/lb-wrapper` for an explanation of how this works.  It refines:

- PR #87
- PR #88

My apologies the PR below (89) might need to be rebased, after this PR #93 is hopefully merged soon:

- PR #89

Tangentially related:

- PR iiab/iiab#3697

### CODE REVIEW

**Comment by holta** [comment]
FYI @deldesir and @EMG70 this PR was tested to show live (real-time) display of "percentage download progress" in the web front-end's "Tasks" view.

While carefully avoiding the pollution of `/var/log/calibre-web.log` and `/var/log/xklb.log` — as subprocesses' STDERR and STDIN streams were polluting log files with duplicate/triplicate (and definitely ugly!) messages.

So hopefully this PR (or similar) can be merged in short order if @deldesir's testing reconfirms this is on the right track! 🛤️

File: scripts/lb-wrapper
Comment (deldesir) [inline scripts/lb-wrapper:1]: Elegant routing!

File: cps/tasks/download.py
Comment (deldesir) [inline cps/tasks/download.py:1]: I confirm this regex works: https://regex101.com/r/47Eiu3/1

**Review by deldesir — APPROVED** [review]
Tested and ready for merge.

File: cps/tasks/download.py
Comment (deldesir) [inline cps/tasks/download.py:1]: Looks good to me

**Comment by holta** [comment]
@deldesir please feel free to improve on these REGEX's (preferably keeping all 3 aligned!) if there's a specific need ✅

**Comment by holta** [comment]
Many Thanks to @EMG70 who reconfirmed — on Lines 1035-1316 here:

https://sprunge.us/1D9ieU?en#n-1035

**Comment by holta** [comment]
I forgot to mention:

This PR also changed log label "Warning" to "Debug" in `/var/log/xklb.log`

As discussed earlier:

- https://github.com/iiab/calibre-web/pull/87#issuecomment-1887855442

**Pull request merged**